site/styles/_carousel.scss

```diff
@@ -13,6 +13,44 @@ s72-carousel {
   @include media-breakpoint-up(lg) {
     height: var(--carousel-height-lg);
   }
+
+  .carousel-video-btns {
+    position: absolute;
+    display: flex;
+    flex-direction: row;
+    align-items: center;
+    height: 30px;
+    bottom: 100px;
+    right: 50px;
+    z-index: 99999;
+
+    #muteBtn {
+      border: none;
+      padding: 0;
+      background: transparent;
+      svg {
+        fill: white;
+        width: 26px;
+        height: 26px;
+      }
+    }
+
+    #trailerBtn {
+      height: 26px;
+      width: 26px;
+      margin: 10px;
+      border: none;
+      background-color: white;
+      -webkit-mask-image: url('/images/icons/video-popup.png');
```

Comment:
prolly better to just use inline png, was just a quick copy paste from the old mute button i had to poc a pop out trailer cta. didnt want to spend more time on it as likely to remove.